addons/spreadsheet/static/src/global_filters/plugins/global_filters_core_plugin.js

```diff
@@ -210,6 +266,12 @@ export class GlobalFiltersCorePlugin extends spreadsheet.CorePlugin {
      */
     import(data) {
         for (const globalFilter of data.globalFilters || []) {
+            if (globalFilter.type === "text" && globalFilter.rangeOfAllowedValues) {
+                globalFilter.rangeOfAllowedValues = this.getters.getRangeFromSheetXC(
+                    "", // there's no default sheet, global filters are cross-sheet
```

Comment:
will end up with traceback if for whatever reason there's no sheet name in the XC :x

Reply:
👍  that's a caveat in the getter

Reply:
https://www.odoo.com/web#id=3578461&cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form